src/ReadVTK.jl

```diff
@@ -443,6 +459,54 @@ function get_data(data_array::VTKDataArray{T,N,<:FormatAppended}) where {T,N}
 end
 
 
+"""
+    get\\_data\\_reshaped(data_array::VTKDataArray; cell_data=false)
+
+Retrieve actual data from a `VTKDataArray` and reshapes them as 1D, 2D or 3D arrays, in case we deal with structured grids.
+Note that vectors or tensors will have their components stored in the first dimension of the array. 
+As there is no way to automatically tell from the VTK file format whether it is a tensor, the user has to reshape this accordingly.
+
+"""
+function get_data_reshaped(data_array::VTKDataArray{T,Ncomponents}; cell_data=false) where {T,Ncomponents}
+
+  data = get_data(data_array);
+
+  # Retrieve size of grid
+  N   = get_local_size(data_array.vtk_file.xml_file, data_array.vtk_file.file_type, cell_data)
+  
+  # reshape 
+  if Ncomponents==1
+    data_reshape = reshape(data, N...)
+  else
+    data_reshape = reshape(data, Ncomponents, N...)
+  end
+
+  return data_reshape
+ 
+end
```

Comment:
```suggestion
  # Retrieve size of grid
  local_size = get_local_size(data_array.vtk_file.xml_file, data_array.vtk_file.file_type, cell_data)
  
  # reshape 
  if N == 1
    data_reshaped = reshape(data, local_size...)
  else
    data_reshaped = reshape(data, N, local_size...)
  end

  return data_reshaped
end
```
As above, use `N` for the parameter and `local_size` for the variable name.